### DISCUSSION TITLE
AmberELEC - 20230203 Panda Conspiracy

### DISCUSSION THREAD

**Discussion by dhwz** [351ELEC/351ELEC#948] [General]
# Release Notes
### What's new?
* Newly added Libretro cores (all devices):
	* **flycast 2021** (Sega Dreamcast / Naomi, Atomiswave)
	* **MAME 2016** (0.174 rom set)
	* **beetle_pce** (NEC Engine / PC Engine CD / TurboGrafx 16 / TurboGrafx CD / SuperGrafx)
	* **DOSBox Core** (MS-DOS)
	* **duckstation** (Sony Playstation)
	* **fake-08** (Pico-8)
	* **ep128emu** (Amstrad CPC, ZX Spectrum, Enterprise 64/128, Videoton TVC)
	* **emuscv** (Epoch Super Cassette Vision)
	* **jaxe** (CHIP-8)
	* **wasm-4** (WASM-4)
	* **lowresnx** (LowRes NX)
	* **mojozork** (Z-Machine)
* Newly added Libretro cores (RG552 only):
	* **bsnes** (Nintento Game Boy / Game Boy Color / MSU-1 / SNES / SFC / Satellaview)
	* **beetle-snes** (Nintento  SNES / SFC / Satellaview)
 	* **bsnes2014 performance** (Nintento SNES / SFC / Satellaview)
 	* **bsnes2014 balanced** (Nintento SNES / SFC / Satellaview)
 	* **bsnes mercury performance** (Nintento SNES / SFC / Satellaview)
 	* **bsnes mercury  balanced** (Nintento SNES / SFC / Satellaview)
	* **mesen** (Nintento NES / Famicom)
	* **mesen-s** (Nintento Game Boy / Game Boy Color / MSU-1 / SNES / SFC / Satellaview)
* Added new options and some new supported cores for  **RetroRun**
* Added an utility to set the time offline
* Added an utility to migrate **Libretro PPSSPP core** save files to its new location

### What has changed?
* Updated **RetroArch** to v1.14.0
* Updated **SDL2** to v2.26.2
* Updated most **Libretro cores** and **standalone emulators** to their latest version
* Updated **MAME - Current core** (0.251 rom set)
* Updated **RetroRun** for better system performance
* Removed **DOSBox SVN core** in favor of **DOSBox Core**
* Some improvements were done to **RetroArch**'s GUI for RG552 (Battery support + proper UI scaling)
* Set default deadzone for **Raze**, **ECWolf**, **Mupen64plus standalone**
* Added m3u support for mplayer
* Updated community shaders
* **Libretro PPSSPP core** save files are now stored in the **PPSSPP standalone** folder `/roms/gamedata/ppsspp`
* General system stability and performance improvements to enhance the user's experience

### Fixes:
* Fixed SDL2 cursor issue on 640x480+ devices
* Fixed anti-piracy detection on some PSP games
* Fixed issue on the 351P where the audio was too low
* Fixed issue with volume control not working
* Fixed skip/back on mplayer with the 351P
* Fixed performance regression on yabasanshiro standalone emulator

### Things to do after an upgrade:
* If you've used the **Libretro PPSSPP core** before, move your save files with the **PPSSPP migration tool** to its new location.

**IMPORTANT NOTE**: There are **four different images** below, one for the **RG351P/M**, **RG351V**, **RG351MP** and **RG552**! 

**If you download the incorrect image for your device, it will not boot!**  If you are unsure, use the the following links:
**New Installations** (`.img.gz`):  **[RG351P/M](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351P.aarch64-20230203.img.gz)** |  **[RG351V](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351V.aarch64-20230203.img.gz)** |  **[RG351MP](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351MP.aarch64-20230203.img.gz)** | **[RG552](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG552.aarch64-20230203.img.gz)**
**Upgrades** (place in `/storage/roms/update`): **[RG351P/M](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351P.aarch64-20230203.tar)** |  **[RG351V](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351V.aarch64-20230203.tar)** |  **[RG351MP](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG351MP.aarch64-20230203.tar)** | **[RG552](https://github.com/AmberELEC/AmberELEC/releases/download/20230203/AmberELEC-RG552.aarch64-20230203.tar)**

<hr /><em>This discussion was created from the release <a href='https://github.com/AmberELEC/AmberELEC/releases/tag/20230203'>AmberELEC - 20230203 Panda Conspiracy</a>.</em>

**Comment by RahaanDev** [comment]
Thank at the team for this Excellent Custom Firmware :)

**Comment by mickael28** [comment]
Do you guys think any of this images could be flash into an SD card for the new Powkiddy RGB30 model? 

Or if not atm, do you know if we might have a build available for that model at some point in the future?

**Reply by dhwz** [reply]
no

no

**Comment by gafar264** [comment]
hello .anyone know what is username and password

**Comment by LuskieAraujo** [comment]


**Reply by LuskieAraujo** [reply]
o arquivo .tar a ser baixado, é AmberELEC-RG351P.aarch64-20230203.tar

**Comment by gpete** [comment]
I've been enjoying this firmware for 2 years. Is this the last release then? The pre-release seems to get contributions regularly but why not the main release?

**Reply by dhwz** [reply]
What exactly is your problem? If you cannot wait just switch to the prerelease branch in update menu.

**Reply by gpete** [reply]
I'm fine waiting but the problem I've been running into recently is RetroAchievements no longer working because the cores on this release are too old. I'll give the pre release another try, but last time I did I ran into several issues

**Reply by Dattebayooooo** [reply]
> I'm fine waiting but the problem I've been running into recently is RetroAchievements no longer working because the cores on this release are too old. I'll give the pre release another try, but last time I did I ran into several issues

Hey man,
Idk If code base is open source.
You can check other roms to understand how it's working and update the cores

**Reply by dhwz** [reply]
> I'm fine waiting but the problem I've been running into recently is RetroAchievements no longer working because the cores on this release are too old. I'll give the pre release another try, but last time I did I ran into several issues

Then just update to prerelease branch, it won't be different if I release it as official release. Unless you've reported your "issues" but then they would have already been fixed in prerelease branch.
Prerelease images are also stable images, just in case you think that would not be the case.

@Dattebayooooo
That actually won't work without rebuilding the whole image.
